crowbar_framework/Gemfile

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-source "https://rubygems.org"
+source "http://rubygems.org"
```

Comment:
It's just because SLE11 is broken. Due to the move is planned for the next sprint I'm fine with waiting.